### ISSUE TITLE
Fix mobile search menu to add `role="search"` to the `form` element

### ISSUE THREAD

**Issue by shoreline-chrism** [shorelinemedia/total-tweaker#6]
The search form in the mobile menu doesn't have an `role` attribute to indicate it's a search form.  

![Screenshot 2022-09-27 141556](https://user-images.githubusercontent.com/9121766/192605049-83dd909d-a238-4ab7-b925-7b46ee92a3ca.jpg)
